rusoto/services/cloudhsm/src/generated.rs

```diff
@@ -560,6 +679,38 @@ pub struct RemoveTagsFromResourceResponse {
     pub status: String,
 }
 
+#[doc="<p>Specifies the type of subscription for the HSM.</p> <ul> <li><b>PRODUCTION</b> - The HSM is being used in a production environment.</li> <li><b>TRIAL</b> - The HSM is being used in a product trial.</li> </ul>"]
```

Comment:
Good example of why we can't just rely on the definitions being enough to use as the only option. (note the missing `Trial` variant) 😄